dataScrape/src/pokemonTableUpdate.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+class pokemonTableUpdate
+{
+    private $db;
+    private $pokemonDataList;
+
+    /**
+     * pokemonTableUpdate constructor.  Opens pokedex db connection
+     *
+     * @param array $pokemonDataList contains an array for each pokemon
+     */
+    public function __construct(array $pokemonDataList)
+    {
+        $this->db = new PDO('mysql:dbname=pokedex_hb;host=127.0.0.1', 'root');
```

Comment:
Does not work on machines with virtualbox.

Reply:
Addressed in b642512.